src/fixup.js

```diff
@@ -297,6 +297,26 @@
     request.send();
   }
 
+  /* Dark mode toggle */
+  const darkCss = document.querySelector('link[href="dark.css"]');
+  if (darkCss) {
+    const btn = document.createElement('button');
+    btn.id = 'theme-toggle';
+    btn.innerText = "Toggle dark mode";
+    document.body.appendChild(btn);
```

Comment:
Adding directly to the body doesn't feel right. I wonder if we should start some kind of `menu` or something..

Reply:
yeah, I couldn't decide where that button should go. @fantasai, any suggestion on that button? The style could definitely be improved as well.

Reply:
Looking at how others do this, I think we need a UI component that can be in one of 3 states... "light", "dark", "auto". 

The best I've seen is how Apple does it (of course): 

![Screen Shot 2021-06-08 at 1 03 40 pm](https://user-images.githubusercontent.com/870154/121116469-f3a12180-c859-11eb-84f4-10d6c205e7ff.png)

They use a `<input type=radio>`, which I think makes sense. 

It also means we don't need to hide the stylesheet. We can just include it as "auto", and then let the use decide which mode they explicitly prefer.

Reply:
If you've like to see how it works at Apple, scroll to the bottom of https://developer.apple.com

Reply:
Thanks @marcoscaceres, I'm actually not a big fan of disabling the stylesheet also but I wasn't sure if "duplicating" the style between the different `prefers-color-scheme` was OK, which, AFAIK, is the only alternative to switch between the schemes.
I've just submitted a version using a `body` data attribute to specify the scheme. I also tried to reproduce the style of the buttons from Apple.

Reply:
I think disabling the style sheet or modifying its 'media' attribute is a perfectly reasonable way to handle this, fwiw. Why wouldn't it be?

Reply:
I don't have a strong opinion on disabling a stylesheet. I don't recall seeing this somewhere else so it seems a bit strange to me but I don't inspect all the pages I visit :)
I've submitted a new version which disables the dark stylesheet in light mode is selected. It also includes @marcoscaceres refactoring:
https://github.com/w3c/tr-design/pull/243/commits/101f56929151e563ee39cc8edabbf13d8d1a413d

Reply:
Looking pretty cool! We probably want to set the background though:   
![image](https://user-images.githubusercontent.com/870154/121993743-5f016b00-cde7-11eb-938d-8af693731c8c.png)

I'm also a bit unsure about having it persistently on screen. 

Trying to figure out where it could live... maybe, on mobile, it should be part of the ToC sidebar?